frontend/src/static/js/v_summary.js

```diff
@@ -278,6 +278,22 @@ window.vSummary = {
       return repo.location;
     },
 
+    getReportIssueLink(stackTrace) {
+      return 'https://github.com/reposense/RepoSense/issues/new?title=' +
+          encodeURI('Unexpected error with RepoSense version ') + window.app.repoSenseVersion + '&body=' +
+          encodeURI(stackTrace);
```

Comment:
The `encodeURI('Unexpected error with RepoSense version ') + window.app.repoSenseVersion` and `encodeURI(stackTrace)` can be split into `getReportIssueTitle` and `getReportIssueMessage` respectively for usability (also single point of modification).